docker/pdo_client.dockerfile

```diff
@@ -54,7 +57,7 @@ ENV PDO_LOG_LEVEL=${PDO_LOG_LEVEL}
 
 # copy the source files into the image
 WORKDIR /project/pdo
-COPY --chown=${UNAME}:${UNAME} repository /project/pdo/src
+COPY --chown=${UNAME}:${UNAME} repository ${PDO_SOURCE_ROOT}
 
```

Comment:
same. there is no reason to make pdo_source_root configurable.

Reply:
see aboe